### ISSUE TITLE
A11y_SQL Azure Data Studio_Extensions - install_Screen reader: After invoking "Next" button Screen Reader is not announcing "Headings" of "Data-tier Application Wizard" section.

### ISSUE THREAD

**Issue by PadminiGariki** [dotnet/maui#16820]
### Description

[Check out Accessibility Insights!] - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### GitHub Tags:
#A11yMAS; #A11ySev3; #A11yTCS; #DesktopApp; #BM_SQL Azure Data Studio_Win32_Aug2023; #SQL Azure Data Studio; #WCAG1.3.1; #ScreenReader; #Win32; #Benchmark; #FTP; #STP; #Element:Heading;

### Environment Details:​
Application Name: Sql Azure Data studio
Version: 1.45.1
OS: Win11

### Steps to Reproduce

1.Launch Azure Data Studio.
2.Connect to server.
3.Navigate to Extensions from left menu.
4.Search & install Dacpac extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right-click a database and select Data-tier application wizard from the context menu.
8.Select first radio button.
9. Navigate till Next button 
10. Now verify that NVDA/Narrator is not announcing "Headings" of "Data-tier Application Wizard" section.

### Actual Results:
After invoking "Next" button Screen Reader is not announcing "Headings" of "Data-tier Application Wizard" section.
**Note:**
**1.Similar issue observed with chrome+ NVDA.**

### Expected Results:
After invoking "Next" button Screen Reader should announce "Headings" of "Data-tier Application Wizard" section.

### User Impact:
Screen reader user will not be able to know "Headings" of "Data-tier Application Wizard" section.. If NVDA/Narrator not announcing.

### Attachment:

[SR not announcing the heading.webm](https://github.com/dotnet/maui/assets/142479541/21ebda08-6b63-491c-bae8-474fecdb89c3)

[SR not annoucning heading after next button NVDA.webm](https://github.com/dotnet/maui/assets/142479541/94a8833c-562b-41ce-b70d-2237fcf47463)

### Version with bug

7.0.92

### Is this a regression from previous behavior?

No, this is something new

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

**Comment by PureWeen** [comment]
Wrong Repository

**Comment by PadminiGariki** [comment]
Sorry for inconvenience. I give correct Repository.
